custom/icds_reports/tasks.py

```diff
@@ -161,13 +163,26 @@ def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
         monthly_dates.append(first_day_next_month - relativedelta(days=1))
 
     monthly_dates.append(date)
-
     db_alias = get_icds_ucr_db_alias()
     if db_alias:
-        with connections[db_alias].cursor() as cursor:
-            _create_aggregate_functions(cursor)
+        test_citus = settings.DATABASES.get(CITUS_MASTER)
+        if test_citus:
+            g = Group()
+            with connections[db_alias].cursor() as cursor:
+                g.spawn(_create_aggregate_functions, cursor)
+            with connections[CITUS_MASTER].cursor() as cursor:
+                g.spawn(_create_aggregate_functions, cursor)
+            g.join()
```

Comment:
getting rid of celery?

Reply:
😏

Reply:
this does seem a bit unnecessary since the time it takes to create the functions should be small